new-lamassu-admin/src/components/editableTable/Row.js

```diff
@@ -44,6 +45,13 @@ const ActionCol = ({ disabled, editing }) => {
     resetForm()
   }
 
+  const [deleteDialog, setDeleteDialog] = useState(false)
+
+  const onConfirmed = () => {
+    onDelete(values.id)
+    setDeleteDialog(false)
```

Comment:
What happens if the delete fails? You can simulate this by shutting down the server prior to confirming. 

The correct thing to do would be to only close the popup if the delete is successful, else show a error message on it. 

This might get pretty tricky on the editable table, so try on the other examples first.